view/templates/content/image_grid.tpl

```diff
@@ -0,0 +1,14 @@
+<link rel="stylesheet" href="view/theme/frio/css/image_grid.css" type="text/css" media="screen"/>
```

Comment:
2 things:
- Please use `Page->registerStylesheet` instead of a `<link>` tag.
- If this CSS stylesheet is used no matter the theme, please move its content to `view/global.css`